test/layout/selectors.js

```diff
@@ -0,0 +1,33 @@
+const { ethers } = require('hardhat');
+const { expect } = require('chai');
+
+describe('Functions layout', function () {
+  it('compare selectors of proxy and upgradable contracts', async function () {
+    // proxy contracts
+    const contractsToCompare = [
+      'NXMaster',
+      'Governance',
+      'MemberRoles',
+      'ProposalCategory',
+      'LegacyPooledStaking',
+      'TokenController',
+      'LegacyGateway',
+      'IndividualClaims',
+      'YieldTokenIncidents',
+      'Assessment',
+      'Cover',
+      'CoverMigrator',
+    ];
+    const { interface: proxyInterface } = await ethers.getContractFactory('OwnedUpgradeabilityProxy');
+    const selectorsToCompare = ['proxyOwner', 'transferProxyOwnership', 'upgradeTo'].map(func =>
+      proxyInterface.getSighash(func),
+    );
+
+    for (const contract of contractsToCompare) {
+      const { interface: contractInterface } = await ethers.getContractFactory(contract);
+      for (const sigHash of selectorsToCompare) {
```

Comment:
have you tried to generate a clash see if it catches it?

If that's the case lgtm

Reply:
Locally I added `proxyOwner` function to the NXMaster to check if it will catch it, and it does. So we are good there.